sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClientIntegrationTest.java

```diff
@@ -406,6 +410,15 @@ void createSubscriptionWithForwarding(HttpClient httpClient) {
             .setForwardDeadLetteredMessagesTo(forwardToTopic);
 
         // Act & Assert
+        if (!interceptorManager.isPlaybackMode()
+            && !client.getTopicExists(topicName).block()) {
```

Comment:
instead of making two network calls you can just handle the ResourceExistsException.

```java
        if (!interceptorManager.isPlaybackMode()) {
            client.createTopic(topicName)
                .onErrorResume(ResourceExistsException.class, error -> Mono.empty())
                .block(TIMEOUT);

            client.createTopic(forwardToTopic)
                .onErrorResume(ResourceExistsException.class, error -> Mono.empty())
                .block(TIMEOUT);
        }
```

Reply:
According to your suggestion, now fixed in the new version.